opennms-base-assembly/src/main/filtered/etc/events/AlarmChangeNotifierEvents.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<events xmlns="http://xmlns.opennms.org/xsd/eventconf">
+  <event>
+    <uei>uei.opennms.org/plugin/AlarmChangeNotificationEvent/AlarmDeleted</uei>
+    <event-label>Alarm Change Notification Event : Alarm Deleted</event-label>
+    <descr>
+    &lt;p&gt;Alarm %parm[alarmid]% Deleted&lt;p&gt;
+    
+    &lt;p&gt;Deleted Alarm Values&lt;p&gt;
+    &lt;p&gt;%parm[oldalarmvalues]%&lt;p&gt;
+    </descr>
+    <logmsg dest="logndisplay">
```

Comment:
The alarm change notifier is a stand alone module which adds value by providing alarm change events which users can use to see who or what changed an alarm on a device. 
By default the events are persisted so that  the alarm life cycle can be read from the OpenNMS event database.  These events could be used in reports or searched for by users. The events also show that the alarm change notifier is working when it is deployed. Users can modify the alarm change event definitions to donotpersist if they find them annoying but I would prefer them to be persisted by default.